src/plasoscaffolder/bll/services/file_handler.py

```diff
@@ -59,29 +59,25 @@ def copy_file(self, source, destination):
     copyfile(source, destination)
     return destination
 
-  def create_or_modify_file_with_content(self, source, content):
+  def create_or_modify_file_with_content(self, source: os.path, content: str):
     self.__create_folder_for_file_path_if_not_exist(self, source)
     self._add_content_to_file(source, content)
 
   @staticmethod
-  def add_content(source, content, exists):
+  def add_content(source: os.path, content: str) -> os.path:
     """ Add content to a file and create file if non existing
 
     :param source: The path of the file to edit.
     :param content: The content to append to the file.
     :return: The path of the edited file.
     """
-    if exists:
-      with open(source, "a") as file:
-        file.write("\n" + content)
-    else:
-      header = "# -*- coding: utf-8 -*-"
-      with open(source, "a+") as file:
-        file.write(header + "\n" + content)
+    with open(source, "a") as file:
```

Comment:
file is a reserved Python keyword, the language allows it, but maybe use file_object instead